src/software/thunderscope/replay/proto_player.py

```diff
@@ -120,7 +123,7 @@ def load_replay_chunk(replay_chunk_path):
         return cached_data
 
     @staticmethod
-    def unpack_log_entry(log_entry):
+    def unpack_log_entry(log_entry: str) -> (float, Message, Message):
```

Comment:
The second return value is the proto type/class. Based on [this article](https://adamj.eu/tech/2021/05/16/python-type-hints-return-class-not-instance/), we should do type hints as follows:
```suggestion
    def unpack_log_entry(log_entry: str) -> (float, type[Message], Message):
```

Reply:
fixed

Reply:
this does not work, it leads to errors

Reply:
nevermind, I have added this